kolla/defaults.yml

```diff
@@ -108,6 +108,8 @@ glance_backends:
     enabled: "{{ glance_backend_swift | bool }}"
 
 glance_allow_anonymous_access: no
+glance_property_protections_file: "{{ node_custom_config }}/glance/property_protections.conf"
+glance_openstack_user: "{{ keystone_admin_user }}"
```

Comment:
is this the way to reference the admin user we want to restrict this metadata field to?

Reply:
if we specify a policy, this can be left out.

worth revisiting later if we want more customization. For now using the "default admin user" is what we do implicitly